index.js

```diff
@@ -229,3 +245,5 @@ module.exports = fp(underPressure, {
   fastify: '3.x',
   name: 'under-pressure'
 })
+
+Object.assign(module.exports, { TYPE_EVENT_LOOP_DELAY, TYPE_EVENT_LOOP_UTILIZATION, TYPE_HEALTH_CHECK, TYPE_HEAP_USED_BYTES, TYPE_RSS_BYTES })
```

Comment:
Please use  explicit `module.exports.SOMETHING =` statements, so that they can be used with native esm.

Reply:
done